src/api/tree.ts

```diff
@@ -1,10 +1,13 @@
+import { LabelInformation } from '../multi-level-checkbox-editor/node-label';
+
 interface BaseNode<T extends BaseNode<T, V>, V> {
     id: string;
     label: string;
     value?: V;
     classes?: string;
     children?: Array<BaseNode<T, V>>;
     isHighlighted?: boolean;
+    labelInformation?: LabelInformation;
 }
 
```

Comment:
What is it used for? Seems unused.

Reply:
No this is the new structure that has been added